pyccl/halos/profiles/einasto.py

```diff
@@ -37,16 +38,23 @@ class HaloProfileEinasto(HaloProfileMatter):
             concentration-mass relation to use with this profile.
         truncated (:obj:`bool`): set to ``True`` if the profile should be
             truncated at :math:`r = r_\\Delta`.
+        projected_integrate (:obj:`bool`): set to ``True`` to calculate the
+            projected profile with numerical integration.
```

Comment:
Can we call this `projected_quad` or something like that? Just to make it clear that this is as opposed to FFTLog.

Reply:
Yes, I was thinking about the naming too.